reference/6/Microsoft.PowerShell.Core/About/about_Logging.md

```diff
@@ -0,0 +1,144 @@
+---
+ms.date:  2017-12-04
+schema:  2.0.0
+locale:  en-us
+keywords:  powershell
+title:  about_Logging
+---
+
+# About Logging
+
+## Short Description
+
+PowerShell logs details of PowerShell operations, such as starting
+and stopping the program engine and starting and stopping PowerShell
+providers.  it will also log details about PowerShell commands.
```

Comment:
`It` needs to be capitalized.  I wonder if it's better to organize this paragraph as bullet points for examples of what gets logged.

Reply:
I'll fix the 'it' but I think the bullet item approach may imply completeness.